docs: Add mandatory modules documentation to the migration-guide.md (#128)
kyma-project/modulectl: docs/contributor/migration-guide.md

7. **Mandatory ModuleTemplates**

>    - Mandatory modules should also be created using `modulectl`. Just add the `mandatory: true` to the module-config. If multiple ModuleTemplates for the same mandatory module get deployed to the Kyma Control Plane, lifecycle-manager picks up the ModuleTemplate with the highest version.

Reviewer: ```suggestion
   - Mandatory modules can be created using `modulectl` by setting **.mandatory** to `true` in the module-config. If multiple ModuleTemplates for the same mandatory module get deployed to the Kyma Control Plane, lifecycle-manager picks up the ModuleTemplate with the highest version.
```
